platform/ext/target/nordic_nrf/nrf5340/target_cfg.c

```diff
@@ -48,7 +59,7 @@ const struct memory_region_limits memory_regions = {
         (uint32_t)&REGION_NAME(Load$$LR$$, LR_VENEER, $$Base),
 
     .veneer_limit =
-        (uint32_t)&REGION_NAME(Load$$LR$$, LR_VENEER, $$Limit),
+        (uint32_t)&REGION_NAME(Load$$LR$$, LR_VENEER, $$Limit) - 1,
```

Comment:
I think this is not correct, @oyvindronningstad because the veneer_limit is used in the MPU configuration, which _expects_ that the veneer_limit points to the address _right after_ the end of the section. If you see the MPU region configuration, the function subtracts 1 byte from the address, so if you subtract one more here, it will not be accurate.

(TBH, this is a bad design from the tf-m code)

For now, I guess you can do the -1 directly in the spu CFG function

Reply:
veneer_limit is used only one place, in `spu_regions_flash_config_non_secure_callable()`, but I can change it in the parameter list instead if you prefer that.

Reply:
Done

Reply:
That's exactly what I meant to say